src/Sinks/SinkBase.php

```diff
@@ -4,6 +4,8 @@
 
 use Illuminate\Support\Carbon;
 use Ragnarok\Sink\Models\SinkFile;
+use League\CommonMark\GithubFlavoredMarkdownConverter;
```

Comment:
composer.json dependency is probably required for this?

Reply:
CommonMark is a dependency of Laravel Framework. Might be needed in ragnarok-sink for isolated unit-testing?
As long as ragnarok-sink runs as a laravel package in a laravel installation League\CommonkMark\GithubFlavoredMarkdownConverter is available. 

I'm no expert in Laravel though, so maybe there should be a reference to league/commonmark in composer.json for ragnarok-sink?